meetings/2024-01-31.md

```diff
@@ -41,28 +49,95 @@
 ## Topics
 
 ### Co-chairs Rotation Schedule
-URL https://github.com/solid/specification/discussions/618
+URL: https://github.com/solid/specification/discussions/618
+
+### Special Topic Meetings
+URL: https://github.com/orgs/solid/projects/16/views/1
 
 ### WIP Implementation Feedback
 * VB: We'll allocate some time for implementation feedback or interest to implement. Links to products/projects and demos welcome.
+* Theo: FedCM, will demo next week with Jan. 
+
+### WG Charter Update
+* Proposed by PAC
+* PAC: initial charter rejected. W3C council convened. Recommendation is to ask for permission to submit new charter, because of remarks, including from the TAG. It was considered that it's not enough to just tweak the charter and a new one is needed. We don't know when to expect a response. As soon we we get the response we want to restart the process. PAC reached out to CG chairs, how to do that in a way that allows everybody to chime in. The question to the group is how to process, PA would like to be faster than just waiting for the weekely meetings. Can we decide how many days and how many approvals we need per PR, without waiting for the CG meeting.
+* SC: looking at the strength of objections is important. Key objections were the proposed chairs (the majority of objections from AC) accounting for 30% of the AC objections. There were other issues about scope and deliverable dependencies. The group since then agree that that's something we need feedback from W3C, not something that we'll be able to solve it. Everything else is relatively marginal. There are things we could do strategically that address the problem space. There were other suggestions on the mailing lists. We could say: these are some specs that we used as input, the group is open to other specs incubated as per W3C recommendation on incubation re open space. This is similar to how Annotation WG and Social Web WG charters. I am still trying to figure out (there another link below) that's flagged in this charter draft. Again, the proposed chairs the primary reason for objections. The rest of the issues are marginal and were expected.
```

Comment:
```suggestion
* PAC: Initial charter rejected. W3C council convened. Recommendation is to ask for permission to submit new charter, because of remarks, including from the TAG. It was considered that it's not enough to just tweak the charter; a new one is needed. We don't know when to expect a response. As soon we we get the response, we want to restart the process. PAC reached out to CG chairs, how to do that in a way that allows everybody to chime in. The question to the group is how to process; PA would like to be faster than the weekly meetings. Can we decide how many days and how many approvals we need per PR, without waiting for the CG meeting?
* SC: Looking at the strength of objections is important. Key objections were the proposed chairs (the majority of objections from AC) which accounted for 30% of the AC objections. There were other issues about scope and deliverable dependencies. The group has agreed since then that we need feedback from W3C on that; it's not something that we'll be able to solve. Everything else is relatively marginal. There are things we could do strategically that address the problem space. There were other suggestions on the mailing lists. We could say: these are some specs that we used as input; the group is open to other specs incubated per W3C recommendation on open space. This is similar to how Annotation WG and Social Web WG were chartered. I am still trying to figure out (there another link below) what's flagged in this charter draft. Again, the proposed chairs were the primary reason for objections. The rest of the issues are marginal and were expected.
```